### ISSUE TITLE
Knitr Engine throws error when code chunk contains comment

### ISSUE THREAD

**Issue by rcst** [rcst/rim#44]
Example from an altered `doc/index.Rmd`:

```
laplace(x) := (2*b)^-1 * exp(-abs(x - mu)/b);

/* load("abs_integrate")$ */
```

```
Error:                                                                                                                                                                                                               
! Bad expression: only one ;|$ termination expression at a time is allowed                                                                                                                                           
Backtrace:                                                                                                                                                                                                           
  1. rmarkdown::render(input = "docs/index.Rmd", output_file = "index.html")                                                                                                                                         
  2. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)                                                                                                                                              
  3. knitr:::process_file(text, output)                                                                                                                                                                              
  7. knitr:::process_group(group)                                                                                                                                                                                    
  8. knitr:::call_block(x)     
```

**Comment by rcst** [comment]
Comments must not be sent to Maxima and skipped.

**Comment by rcst** [comment]
For the knitr engine, comments could span multiple lines. Requires a re-write of `gather()`.

**Comment by rcst** [comment]
Entirely fixed in latest release 0.7.0., closing this now.
